superagi/tools/duckduckgo_serp_search/duckduckgo_serp_search.py

```diff
@@ -0,0 +1,35 @@
+from typing import Type
+from pydantic import BaseModel, Field
+
+from superagi.helper.duckduckgo_serp import DuckDuckGoSerpApiWrap
+from superagi.tools.base_tool import BaseTool
+from superagi.config.config import get_config
+
+import os
+
+import json
+
+
+class DuckDuckGoSerpSchema(BaseModel):
+    query: str = Field(
+        ...,
+        description="The search query for DuckDuckGo SERP.",
+    )
+
+
+class DuckDuckGoSerpTool(BaseTool):
+    name = "DuckDuckGoSerp"
+    description = (
+        "A tool for performing a DuckDuckGo SERP search and extracting snippets and webpages."
+        "Input should be a search query."
+    )
+    args_schema: Type[DuckDuckGoSerpSchema] = DuckDuckGoSerpSchema
+
+    def _execute(self, query: str) -> tuple:
+        api_key = get_config("SERP_API_KEY")
```

Comment:
Might want to use a different env var name than `SERP_API_KEY` which is currently used for the [Google Serper.dev tool](https://github.com/TransformerOptimus/SuperAGI/blob/3fc82cd3b469aa55fdc1a789f9f366a56670ab00/superagi/tools/google_serp_search/google_serp_search.py#L35)